client/pages/edit-delete-form.jsx

```diff
@@ -60,14 +72,62 @@ class ReservationForm extends React.Component {
       .catch(err => console.error(err));
   }
 
+  handleDelete() {
+    event.preventDefault();
+    const { uniqueCode } = this.state;
+    const data = { uniqueCode };
+    const req = {
+      method: 'DELETE',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(data)
+    };
+    fetch('/api/delete-reservation', req)
+      .then(response => {
+        if (response.ok) {
+          this.setState({ submitted: true });
+        } else {
+          const { ok, status, statusText } = response;
+          this.setState({ errorObject: { ok, status, statusText } });
```

Comment:
It's not good for users to see status codes and technical jargon. Instead, try something more human-friendly.